### ISSUE TITLE
Wayland tearing protocol support

### ISSUE THREAD

**Issue by Jontique** [pop-os/cosmic-comp#248]
Hey.

I was wondering if COSMIC will support the Wayland tearing protocol, at very least for fullscreen applications.

Allowing screen tearing makes a huge difference in reducing input latency in FPS games and games like Osu!

https://wayland.app/protocols/tearing-control-v1

**Comment by Drakulix** [comment]
Yes, it is on the TODO list and we will eventually support it. **But** it's not a very high priority.

**Comment by LucasGGamerM** [comment]
Where can one see the TODO list? Is it implemented on the alpha?

**Comment by Drakulix** [comment]
If it would be implemented this issue wouldn't be open any more.
You can take a look at the current prioritization here: https://github.com/orgs/pop-os/projects/23
Any open issue is part of the hypothetical "TODO" list, but those are the issues we actively working towards at the moment.

**Comment by FinGu** [comment]
Bumping this.. ( would also love for an option to always enable tearing on fullscreen apps )

**Comment by AHuman-Person** [comment]
Chiming in as someone trying to switch to popos from win11. Having like 300 quadrillion years of input latency really makes gaming not worth it on pop os(geometry dash specifically, ever single frame counts, especially on 60hz)

**Comment by gabriele2000** [comment]
> Chiming in as someone trying to switch to popos from win11. Having like 300 quadrillion years of input latency really makes gaming not worth it on pop os(geometry dash specifically, ever single frame counts, especially on 60hz)

Funny: I've never had issues with latency, could be because I have a 120Hz screen, even though I play at 60FPS?
Now I'm really wondering if that's the case, since I've never had latency issues.

**Comment by EmanuelKroell** [comment]
Hi, missing support for tearing is currently (and has been for almost 2 years now) the biggest issue keepinge from using Cosmic or another smithay based compositor like Niri.
I hope that especially with the influx of people who primarily want to play games coming over from windows, tearing would be a higher priority/at least be added to the roadmap at some point.

**Comment by Jontique** [comment]
Hello!
Seeing this had gained some reactions and comments over the years, I am requesting an update on whether if this is still planned. Or perhaps prioritizing it higher, as it is a quite vital for low input latency-dependent games. 

Also adding the Gaming tag on the issue would be great, so it is grouped with the rest of the gaming-related improvements at: https://github.com/pop-os/cosmic-comp/issues?q=is%3Aissue%20state%3Aopen%20label%3Agaming
